scss/civicrm/common/_forms.scss

```diff
@@ -480,6 +379,88 @@
     }
   }
 
+  %responsive {
```

Comment:
why do we have these placeholders lying around? don't we have a folder for them?

Reply:
No, we don't have any directory for placeholders and there are many placeholders scattered around different styles.